### PR TITLE
Add base structure

### PR DESCRIPTION
**Pull request by Karina5005**
Fixes: https://github.com/cloud-labs-infra/github-backup/issues/1

### CODE REVIEW

**Review by bissquit — COMMENTED** [review]
Maybe it could be better to name folder the same as application name ('github-backup') but not `backup` because it looks ambiguously

**Comment by Karina5005** [comment]
I think it's a bad idea to name a folder `github-backup`. At least because modules in python should not contain hyphens. Initially, I thought to split it into two folders `backup` and `restore`, or I can name the folder `github_backup`, but in such case it will not be the same as application name.

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+           coverage report
+           coverage html
+
+[flake8]
+show-source = true
+exclude = .tox/
+          venv/
+
+max-line-length = 180
+import-order-style = google
```
Comment (bissquit) [inline tox.ini:25]: Don't forget to add a new line

**Review by bissquit — APPROVED** [review]
LGTM

**Pull request merged**